packages/gasket-fetch/browser.js

```diff
@@ -0,0 +1,9 @@
+
+exports = window.fetch; // To import fetch from @gasket/fetch
+exports.default = window.fetch; // For TypeScript consumers without esModuleInterop.
+exports.fetch = window.fetch; // To import {fetch} from @gasket/fetch
+exports.Headers = window.Headers;
+exports.Request = window.Request;
+exports.Response = window.Response;
+
```

Comment:
Let's make `AbortController` available from this package in browser/node.

```suggestion
exports.AbortController = window.AbortController;

```